packages/remix/src/index.server.ts

```diff
@@ -118,10 +124,21 @@ export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
 export { withSentry } from './client/performance';
 export { captureRemixErrorBoundaryError } from './client/errors';
 export { browserTracingIntegration } from './client/browserTracingIntegration';
-export { wrapExpressCreateRequestHandler } from './utils/serverAdapters/express';
```

Comment:
Hmm, I guess/fear we have to continue exporting this (as removing this is a breaking change 😬 ) We can make it noop I guess..?

Reply:
Updated 👍